Tasks/SonarQubePostTest/CodeAnalysisFilePathComputation.ps1

```diff
@@ -0,0 +1,275 @@
+$ProjectGuidAndFilePathMap = @{}
```

Comment:
If most of the code has already been reviewed it would help if you pointed out what you've changed so reviewers can focus on those bits.